Fix create-layer function for compleseus

fnussbaum opened the pull request:
Fixes #15913

bcc32 commented:
FWIW, `ivy-read` with these arguments is equivalent to just `completing-read`, so I think the only special case needed is helm, here.

fnussbaum committed:
Fix create-layer function for compleseus

When using ivy, the new completing-read call is equivalent to the previous ivy-read call.

fnussbaum commented:
Good point, thanks!

smile13241324 commented on core/core-configuration-layer.el:
```diff
@@ -715,12 +715,12 @@ layer directory."
+         (layer-path-sel (if (configuration-layer/layer-used-p 'helm)
+                             (helm :sources helm-lp-source
+                                   :prompt "Configuration layer path: ")
+                           (completing-read "Configuration layer path: "
+                                            (append current-layer-paths
+                                                    (list other-choice)))))
```
Are you sure we do not need ivy read here too?

smile13241324 reviewed (changes requested):
Please have a look at my review comment

smile13241324 commented:
Ups missed the last comment
